app/services/audit_results.rb

```diff
@@ -14,12 +14,12 @@ class AuditResults
 
   INVALID_ARGUMENTS = :invalid_arguments
   VERSION_MATCHES = :version_matches
-  ARG_VERSION_GREATER_THAN_DB_OBJECT = :arg_version_greater_than_db
-  ARG_VERSION_LESS_THAN_DB_OBJECT = :arg_version_less_than_db
+  ACTUAL_VERS_GT_DB_OBJ = :arg_version_greater_than_db
+  ACTUAL_VERS_LT_DB_OBJ = :arg_version_less_than_db
   CREATED_NEW_OBJECT = :created_new_object
```

Comment:
`ARG` changes to `ACTUAL` on the left, but `arg` stays `arg` on the right?

Reply:
hahahaha - oops.  thanks.